graphqlapi/src/main/java/io/stargate/graphql/schema/cqlfirst/dml/fetchers/BulkMutationFetcher.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.graphql.schema.cqlfirst.dml.fetchers;
+
+import static io.stargate.graphql.schema.SchemaConstants.ATOMIC_DIRECTIVE;
+import static java.util.stream.Stream.concat;
+
+import graphql.GraphQLException;
+import graphql.language.OperationDefinition;
+import graphql.schema.DataFetchingEnvironment;
+import io.stargate.auth.AuthenticationSubject;
+import io.stargate.auth.AuthorizationService;
+import io.stargate.db.datastore.DataStore;
+import io.stargate.db.datastore.DataStoreFactory;
+import io.stargate.db.query.BoundQuery;
+import io.stargate.db.schema.Table;
+import io.stargate.graphql.schema.cqlfirst.dml.NameMapping;
+import io.stargate.graphql.web.HttpAwareContext;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public abstract class BulkMutationFetcher
```

Comment:
Why is this abstracted out from `BulkInsertMutationFetcher`? Are there plans to have an bulk update mutation?

Reply:
This PR introduces bulk mutations